packages/neutron-sdk/src/interchain_queries/types.rs

```diff
@@ -122,3 +123,13 @@ pub trait KVReconstruct: Sized {
     /// Reconstructs this value from the slice of **StorageValue**'s.
     fn reconstruct(kvs: &[StorageValue]) -> NeutronResult<Self>;
 }
+
+impl KVReconstruct for Uint128 {
```

Comment:
seems to be unused. what's the point of this definition? or am I missing something?

Reply:
1. It's being used here: https://github.com/neutron-org/neutron-sdk/blob/e493cf80ad28f72412b5ebe8bb6c0be9018087e7/contracts/neutron_interchain_queries/src/contract.rs#L347
2. Even it's not being used explicitly, it's nice to have reconstruction method for different types